spack.yaml

```diff
@@ -0,0 +1,33 @@
+# traccc library, part of the ACTS project (R&D line)
+#
+# (c) 2025 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+spack:
+  specs:
+    # Build tools
+    - "cmake@3.26:"
+    - "intel-oneapi-compilers@2024.2.0"
+    - "cuda@12.5"
+    - "gcc@13"
```

Comment:
Is spack able to use this compiler yet? Last I checked this was still pending and waiting for a change to include compilers in the dependency graph.

Reply:
Indeed that is true, the purpose of this is just to make gcc 13 available to the user, rather than to require it be used for compiling e.g. ACTS.